qiita_pet/templates/portals_edit.html

```diff
@@ -0,0 +1,134 @@
+{% extends sitebase.html%}
+
+{%block head%}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+<style type="text/css">
+  .navlist li
+  {
+    display: inline;
+    padding-right: 20px;
+  }
+  .portal-select {
+    width: 15em;
+  }
+</style>
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript">
+function check_submit(action) {
+  //disable submit buttons
+  $("#add-button").prop('disabled', true);
+  $("#remove-button").prop('disabled', true);
+  $("#messages").html("");
+  var error = false;
+  if($("#portal").val() === '') {
+    //No selected portal, so add error message if needed and then don't submit
+    if(!$('#portal-error').length) { $("#errors").append("<li id='portal-error'>Please select a portal</li>"); }
+    error = true;
+  }
+  var data = oTable.$('input').serialize();
+  if(data.length === 0) {
+    //No checked rows, so add error message if needed and then don't submit
+    if(!$('#checkbox-error').length) { $("#errors").append("<li id='checkbox-error'>Please select at least one row</li>"); }
+    error = true;
+  }
+  if(error) {
+    $("#add-button").prop('disabled', false);
+    $("#remove-button").prop('disabled', false);
+    return false;
```

Comment:
`return false;` stops the submission in its tracks. There is no need to return true since it will continue on and submit at the end. Yay javascript conventions.